_search-plugins/search-pipelines/deleting-search-pipeline.md

```diff
@@ -0,0 +1,26 @@
+---
+layout: default
+title: Deleting search pipelines
+nav_order: 30
+has_children: false
+parent: Search pipelines
+grand_parent: Search
+---
+
+# Deleting search pipelines
+
+Use the following request to delete a pipeline.
+
+To delete a specific search pipeline, pass the pipeline ID as a parameter:
+
+```json
+DELETE /_search/pipeline/<pipeline-id>
+```
+{% include copy-curl.html %}
+
+To delete all search pipelines in a cluster, use the wildcard character (`*`):
+
+```json
+DELETE /_search/pipeline/*
```

Comment:
Did we test this as ingest pipeline doesn't have a [doc](https://opensearch.org/docs/1.2/opensearch/rest-api/ingest-apis/delete-ingest/) for it?

Reply:
https://github.com/opensearch-project/OpenSearch/blob/6b8b3efe01a62c221f308a2e3b019d75a7f5ad8a/server/src/main/java/org/opensearch/rest/action/search/RestDeleteSearchPipelineAction.java#L33-L36

Reply:
Of note, the ingest one doesn't include the leading slash. So that probably should be deleted.

Reply:
But the other search pipeline [docs](https://opensearch.org/docs/2.15/search-plugins/search-pipelines/creating-search-pipeline/) do have the leading slash so we should be consistent with those.

Reply:
Ingest does have a leading slash - @owaiskazi19 that link is for 1.2 documentation :)
 https://opensearch.org/docs/latest/ingest-pipelines/delete-ingest/